src/engine.rs

```diff
@@ -105,21 +101,9 @@ mod tests {
     use std::path::Path;
 
     // Arbitrary, used to verify that the runner works as expected.
-    const FUNCTION_SLEEP_DURATION: Duration = Duration::from_millis(42);
     const HELLO_WORLD_MEMORY_USAGE: u64 = 17;
     const MODIFIED_HELLO_WORLD_MEMORY_USAGE: u64 = 42;
 
-    #[test]
-    fn test_runtime_over_threshold() {
```

Comment:
Repeating an earlier comment I made about deleting this because the deterministic WASI context crate makes `sleep` a no-op to match the behaviour in production and we don't have an alternative approach that produces a consistently long running operation.